justfile

```diff
@@ -174,3 +174,15 @@ compile ncpu=num_cpus() *CONFIG='': (configure CONFIG) (build ncpu)
 
 # re-build ldmx-sw and then run a config
 recompFire config_py *ARGS: build (fire config_py ARGS)
+
+# install the validation module
+# `python3 -m pip install Validation/` is the standard `pip` install method.
+# We add `--upgrade` to tell `pip` it should overwrite the package if it already has been
+# installed before which is helpful in the case where someone is updating the code and running
+# the new code within the container. The `--target install/python/` arguments tell `pip`
+# where to install the package. This directory is where we currently store our python modules
+# and is where the container expects them to be. The `--no-cache` argument tells `pip` to
+# not use a cache for downloading any dependencies from the internet which is necessary since
+# `pip` will not be able to write to the cache location within the container.
```

Comment:
The last comment line before a recipe is the one printed with the help message, so I am just reordering this so that the help message is "install the validation python module".

```suggestion
# `python3 -m pip install Validation/` is the standard `pip` install method.
# We add `--upgrade` to tell `pip` it should overwrite the package if it already has been
# installed before which is helpful in the case where someone is updating the code and running
# the new code within the container. The `--target install/python/` arguments tell `pip`
# where to install the package. This directory is where we currently store our python modules
# and is where the container expects them to be. The `--no-cache` argument tells `pip` to
# not use a cache for downloading any dependencies from the internet which is necessary since
# `pip` will not be able to write to the cache location within the container.
# install the python Validation plotting module
```